Prism/Functions/Invoke-Prism.ps1

```diff
@@ -138,23 +130,47 @@ function Invoke-Prism
                     $lockExtension = ''
                 }
 
-                $lockPath = Join-Path -Path ($prismJsonPath | Split-Path -Parent) -ChildPath "$($lockBaseName).lock$($lockExtension)"
-                # private members that users aren't allowed to customize.
-                $config |
-                    Add-Member -Name 'Path' -MemberType NoteProperty -Value $prismJsonPath -PassThru -Force |
-                    Add-Member -Name 'File' -MemberType NoteProperty -Value $prismJsonFile -PassThru -Force |
-                    Add-Member -Name 'LockPath' -MemberType NoteProperty -Value $lockPath -PassThru -Force |
-                    Out-Null
-
                 $ignore = @{ 'ErrorAction' = 'Ignore' }
                 # public configuration that users can customize.
-                # Add-Member doesn't return an object if the member already exists, so these can't be part of the pipeline.
+                # Add-Member doesn't return an object if the member already exists, so these can't be part of one pipeline.
                 $config | Add-Member -Name 'PSModules' -MemberType NoteProperty -Value @() @ignore
                 $config | Add-Member -Name 'PSModulesDirectoryName' -MemberType NoteProperty -Value 'PSModules' @ignore
 
+                $privateModulePath =

```

Comment:
Hmm. I haven't seen that. I did a search for usages of `Resolve` and can't see where this would be happening. Next time you see that error, capture a full stack trace and send it to me: `$Error[x] | Format-List * -Force`.